cli/command/trust/inspect.go

```diff
@@ -0,0 +1,83 @@
+package trust
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/theupdateframework/notary/tuf/data"
+)
+
+func newInspectCommand(dockerCli command.Cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "inspect IMAGE[:TAG]",
+		Short: "Return low-level information about keys and signatures",
+		Args:  cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return inspectTrustInfo(dockerCli, args[0])
+		},
```

Comment:
I believe all other inspect commands support multiple args for inspecting multiple objects at once, so they return a list instead of a single JSON object.

Other inspect commands also use a formatter in `cli/command/formatter` to support the `--format` flag.

I think it might be good to be consistent here.

Reply:
sounds good.  I've added multiple args support.  I'd like to hold off on the `--format` flag for now until we've stabilized the JSON format